rdrf/rdrf/features/steps.py

```diff
@@ -7,17 +7,92 @@
 
 from nose.tools import assert_true, assert_equal
 
+from selenium.webdriver.common.keys import Keys
 from selenium.webdriver.common.alert import Alert
 
 from . import utils
 
+from collections import OrderedDict
+import time
 
 logger = logging.getLogger(__name__)
 
 # Clearing all the aloe step definitions before we register our own.
 STEP_REGISTRY.clear()
 
 
+@step('I try to log in')
+def try_to_login(step):
+    world.browser.get(world.site_url + "login?next=/router/")
+
+
+@step('I try to register as an "([^"]+)" user')
+def try_to_register(step, registry):
```

Comment:
As I said in the Angelman PR - would be better to make this less coarse-grained, with obvious step inputs ( like patient name, email address (username.)